.classpath

```diff
@@ -53,8 +53,7 @@
 	<classpathentry kind="lib" path="/jars/lib/jars/mysql/mysql-connector-java-5.1.22-bin.jar"/>
 	<classpathentry kind="lib" path="/jars/lib/jars/kbase/kidl/kbase-kidl-parser-1409261812-7863aef.jar"/>
 	<classpathentry kind="lib" path="/jars/lib/jars/jsemver/java-semver-0.9.0.jar"/>
-	<classpathentry kind="lib" path="/jars/lib/jars/kbase/handle/HandleManagerClient-160803-08c8da3.jar"/>
-	<classpathentry kind="lib" path="/jars/lib/jars/kbase/handle/HandleServiceClient-160803-b9de699.jar"/>
+	<classpathentry kind="lib" path="/jars/lib/jars/kbase/handle/AbstractHandleClient-1.0.0.jar"/>
```

Comment:
Get rid of the SQL jar here and in build.xml

Reply:
👍